dace/codegen/targets/intel_fpga.py

```diff
@@ -1141,7 +1151,8 @@ def unparse_tasklet(self, sdfg, state_id, dfg, node, function_stream,
                                        locals,
```

Comment:
(comment about line 1090)
Since you're in the Intel FPGA codegen, don't you want to support dtypes.Language.OpenCL as a tasklet language? (and also change the exception raised from "CPU" to "Intel FPGA")

Reply:
thanks!